### PR TITLE
Feat/master/use numerical users

### PR DESCRIPTION
**Pull request by anestos**
This PR has the following changes and should be merged once 9.0.0 is released.
- Remove maintance of images 7.x
- Update Debian based images to bookworm for version 9
- Update Centos based images to Centos Stream9 for version 9
- build OpenSSL 3.0.13 in centos images instead of 1.1.1
- Create numerical user and group 10001:10000

### CODE REVIEW

**Comment by anestos** [comment]
Update to OpenSSL 3.0.13 once it is out.

File: Dockerfile-centos.template [hunk 1]
```diff
@@ -43,17 +43,20 @@ EXPOSE 4466
 
 RUN yum update -y && yum clean all
 
-RUN useradd --system idsvr
+RUN groupadd --system --gid 10000 idsvr && useradd --system --no-create-home --uid 10001 --gid 10000 idsvr
 
 ENV IDSVR_HOME /opt/idsvr
 ENV JAVA_HOME $IDSVR_HOME/lib/java/jre
 ENV PATH $IDSVR_HOME/bin:$JAVA_HOME/bin:$PATH
+ENV LD_LIBRARY_PATH=/usr/local/lib64
```
Comment (luisgoncalves) [inline Dockerfile-centos.template:51]: Is it empty by default, i.e. no need to concat with existing value?

**Review by luisgoncalves — APPROVED** [review]

File: Dockerfile-centos.template (hunk 1 above)
Comment (anestos) [reply Dockerfile-centos.template]: It is not necessary to set this actually, i removed it now. It was there from the previous tries i did where i copied the whole openssl build into `/usr/local/lib64` but i changed that after some testing last week and forgot to remove this.

**Review by gary-archer — COMMENTED** [review]
**BASIC TEST RESULTS**

I did a little testing to help out, and also to better understand this repo - these steps worked fine:

8.7.0 Docker image
-  I ran an Ubuntu build and it worked fine, but see the below README issue
-  I then deployed the image and tested passkey logins
-  I then remoted to the image and ran `id`, with these details: `uid=999(idsvr) gid=999(idsvr) groups=999(idsvr)`

9.0.0 Docker image
-  I ran `VERSION=9.0.0 ./add-release.sh`
-  I then created an `idsvr-9.0.0-linux` folder (though I used the 8.7.0 files)
-  I then deployed the image and tested passkey logins
-  I then remoted to the image and ran `id`, with these details: `uid=10001(idsvr) gid=10000(idsvr) groups=10000(idsvr)`

**MINOR POINT**

- `Dockerfile-rhel.template` does not seem to be updated to a numeric user ID so worth updating that for consistency

**README**

1. The reference `update-images.sh` does not exist and should be renamed to `update-multiplatform-images.sh`

2. This section on **Building a Single Image** did not work for me due to path and TARGETARCH issues. Maybe we can tweak the customer instructions slightly:

Screenshot 2024-02-08 at 18 22 18

**Review by gary-archer — APPROVED** [review]

**Pull request merged**